src/slic3r/GUI/GLCanvas3D.cpp

```diff
@@ -1534,6 +1536,16 @@ ModelInstanceEPrintVolumeState GLCanvas3D::check_volumes_outside_state(ObjectFil
     return state;
 }
 
+void GLCanvas3D::toggle_world_axes_visibility(bool force_show)
+{
+    if (force_show) {
+        m_show_world_axes = true;
+    } else {
+        m_show_world_axes = !m_show_world_axes;
```

Comment:
I think you need to save the value back to app config otherwise the setting will lost once Orca is closed.